addons/web/static/src/js/core/dialog.js

```diff
@@ -128,6 +134,11 @@ var Dialog = Widget.extend({
             });
             self.$footer.append($button);
         });
+        // Adding a ghost button to organize footer buttons
+        // better in full-screen mode on small devices
+        if (this.isMobile) {
+            this.$footer.append($('<button class="btn o_btn_ghost"/>'));
```

Comment:
I don't understand why you need this ghost button. What happens if we remove it ?